CanonicalLtd/snappy-docs
# Add Deprecation Notices

## QA
- Check the copy is ok
- Check the name of the new folder is links properly from the homepage
